File: Compiler/UnitTestSample_1/GlobalVar.cs
using JavaScript;

namespace UnitTestSample_1
{
    public class GlobalVar
    {
        [GlobalVariable]
        private int x;

        private void Step1()
        {
            x = 10;
        }

        private void Step2()
        {
            x++;
        }

        public void Do()
        {
            Step1();
            Step2();
        }
    }
}

File: Compiler/UnitTestSample_1/ToCompare/GlobalVar.js
enyo.kind({
    name: "UnitTestSample_1.GlobalVar",
    kind: "enyo.Object",

    step1 : function(){
        x = 10;
    },

    step2 : function(){
        x++;
    },

    do : function(){
        this.step1();
        this.step2();
    }
});
